components/core/NewWebsitePrototypeHeader.js

```diff
@@ -225,11 +225,10 @@ const STYLES_MENU_OPEN = css`
   }
 `;
 
-export const NewWebsitePrototypeHeader = (props) => {
+const NewWebsitePrototypeHeader = props => {
   const [open, setOpen] = useState(false);
 
-  const viewSourceURL = "https://github.com/filecoin-project/slate";
-  const designSystemURL = "/_/system";
+  const commuityURL = "/community";
```

Comment:
nit: community*

Reply:
Spell check in IDEs is brutal good catch.